webpack.config.js

```diff
@@ -17,16 +20,16 @@ module.exports = {
     resolve: {
         mainFields: ['browser', 'main', 'module'],
         extensions: ['*', '.js'],
+        fallback: {
+            fs: false,
+            os: false,
+            stream: false,
+            util: false,
+            zlib: false,
+        },
```

Comment:
Not sure how webpack 5 deals with Node internals but please check the bundle size between the versions. E.g. `Buffer: false` was there earlier to remove a pretty large `Buffer` polyfill.

Reply:
Almost all nodejs APIs have been removed from webpack 5, and you need to manually add them (like I had to do with process). 

Nonetheless, here's an old vs new comparison (we actually are saving some bytes)
![image](https://user-images.githubusercontent.com/17960496/129334467-6a29966a-9c57-4261-954d-b9beb6501cde.png)

Reply:
Hmm, that looks big. We must have been doing something wrong with this bundle. https://bundlephobia.com/package/apify-client@1.3.0 It's used only for testing so maybe it's not that important, but still.

Reply:
It is now almost 4x smaller (almost..)
![image](https://user-images.githubusercontent.com/17960496/129354491-f3ce0f55-98b8-4b2f-97db-2fd33bb670cf.png)